src/zenml/integrations/kubeflow/flavors/kubeflow_orchestrator_flavor.py

```diff
@@ -206,7 +251,7 @@ def is_remote(self) -> bool:
         """
         if (
             self.kubernetes_context is not None
-            and not self.kubernetes_context.startswith("k3d-zenml-kubeflow-")
+            and not self.kubernetes_context.startswith("k3d-minimal-zenml-")
```

Comment:
This is what I call a "weak" convention, in the sense that we don't document it anywhere and we have no way of enforcing it with code or other validation mechanisms. We can only hope that the user doesn't use a different name for the k3d cluster deployed with the stack recipe. In addition to that, it is also quite obfuscated, as it gives no clear indication that using this prefix is the only thing that makes the orchestrator "local".

I would propose being painfully explicit but simple here by adding a `local` flag to the configuration.